src/cc/Httpd.cpp

```diff
@@ -43,19 +45,54 @@ bool Httpd::start()
         return false;
     }
 
-    m_daemon = MHD_start_daemon(MHD_USE_SELECT_INTERNALLY, static_cast<uint16_t>(m_options->ccPort()), nullptr, nullptr, &Httpd::handler,
-                                this, MHD_OPTION_CONNECTION_TIMEOUT, (unsigned int) 10, MHD_OPTION_END);
+#   ifndef XMRIG_NO_TLS
+    if (m_options->ccUseTls()) {
+
+        m_keyPem = readFile(m_options->ccKeyFile());
+        m_certPem = readFile(m_options->ccCertFile());
+
+        if (m_keyPem.length() == 0 || m_certPem.length() == 0) {
```

Comment:
better: `!m_keyPem.empty() || !m_certPem.empty()`

Reply:
Should be `m_keyPem.empty() || m_certPem.empty()` i hope :D